protocols/encoding/decoder.go

```diff
@@ -0,0 +1,32 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package encoding
```

Comment:
Now that I saw the model versions, maybe for the current Encoder/Decoder a `pbytes` or just bytes is a nice package name. What do you think?

Reply:
Made this bytes. Has potential for conflicting with golang bytes package but I think good enough for now.